test/parallel/test-sqlite-session.js

```diff
@@ -199,12 +197,6 @@ suite('conflict resolution', () => {
   });
 });
 
-test('session related constants are defined', (t) => {
-  t.assert.strictEqual(SQLITE_CHANGESET_OMIT, 0);
-  t.assert.strictEqual(SQLITE_CHANGESET_REPLACE, 1);
-  t.assert.strictEqual(SQLITE_CHANGESET_ABORT, 2);
-});
-
```

Comment:
IMHO we indeed shouldn't check constants against hardcoded values, but we should keep a test that they are defined. How about this?
```js
test('session related constants are defined', (t) => {
  t.assert.strictEqual(typeof SQLITE_CHANGESET_OMIT, 'number');
  t.assert.strictEqual(typeof SQLITE_CHANGESET_REPLACE, 'number');
  t.assert.strictEqual(typeof SQLITE_CHANGESET_ABORT, 'number');
});
```

Reply:
Thanks for your review, @LiviaMedeiros .

I sense this will weaken the test. `SQLITE_CHANGESET_OMIT, for instance, has to be `0`. If it's something else, people might have problems using it, and we probably want the tests to break.

SQLite has [a couple of constants](https://www.sqlite.org/session/constlist.html). I don't think they change very often. If they change in a new version it would be good to assert the new values.

Does that make sense for you?

Reply:
Ah, sure. Since these constants are not Node.js-specific and are defined elsewhere, it would make sense to keep this test as is.